licenses/star LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
those licenses files don't need to be referenced somehow from the pyproject.toml file?